packages/flutter/example/linux/flutter/generated_plugin_registrant.cc

```diff
@@ -6,10 +6,6 @@
 
 #include "generated_plugin_registrant.h"
 
-#include <awesome_notifications/awesome_notifications_plugin.h>
```

Comment:
This seems to be removing or doing some modification on awesome_notifications; this is not clear from the PR title. Could you please make sure the PR title fully reflects the changes in this PR?

Reply:
This class is automatically created by flutter

Reply:
In general, files that start with generated_ are automatically created by flutter

Reply:
Strange, where do these changes come from? The change in the source do not seem to be relates to awesome_notifications?

Reply:
It seems to me we should merge this PR without these changes. If they are unrelated to and not an effect of this PR. If necessary they can be committed in a separate `refactor` PR.

Reply:
If we do not merge these changes that are added or removed automatically by Flutter in this PR, we will have problems in the Windows and Linux versions because `awesome_notifications` has been removed.

Reply:
Yes it has been removed, but not in this PR, right?

Reply:
Yes, then it is better to delete it

Reply:
Alright, could you remove these changes, so we can merge this PR?

Reply:
removed